### ISSUE TITLE
lazy umount for fuse mounts

### ISSUE THREAD

**Issue by cehteh** [ipfs/kubo#2712]
Would adding the -z option (lazy unmounting) make any sense? if something accesses the fuse mounts  and one kills the daemon there are still lingering mounts left behind, preventing a restart.

The only problem I see would be that it could shadow other bugs leaving the filesystem mounted, but all other umount command either do their job, block or report an error.

**Comment by whyrusleeping** [comment]
Yeah... fuse is tricky that way. I'd really like to move towards running the fuse mount as a separate process, independent of the daemon (although it wouldnt function properly without the daemon running).

**Comment by Kubuxu** [comment]
I think we all agreed that blocker for that is UNIX socket local API transport for performance reasons. I might sit down and spec out something about it, for example encoding schema.

**Comment by linas** [comment]
Seems to be a duplicate of #1248 which describes fuse messiness after unclean shutdown of ipfs.
